### PR TITLE
TM-6150-Updated-TED

### PR DESCRIPTION
**Pull request by cagbakwu**

### CODE REVIEW

File: talentmap_api/common/common_helpers.py [hunk 1]
```diff
@@ -251,11 +251,14 @@ def ensure_date(date, utc_offset=0):
...
-    # Format the date as desired
-    formatted_date = parsed_date.strftime("%Y-%m-%d")
+    # Remove the colon in the timezone offset
+    date_str = date[:-3] + date[-2:]
+
+    # Parse the date string to a datetime object
+    date_obj = datetime.strptime(date_str, "%Y-%m-%dT%H:%M:%S%z")
+
+    # Format the datetime object to the desired format
+    formatted_date = date_obj.strftime("%Y-%m-%d")
```
Comment (cagbakwu) [inline talentmap_api/common/common_helpers.py:261]: Updated to match ted date format in Dev1. Tested in Dev1 and passed.

File: talentmap_api/fsbid/services/client.py [hunk 2]
```diff
@@ -309,9 +309,10 @@ def fsbid_clients_to_talentmap_clients(data):
 
 
 def fsbid_clients_to_talentmap_clients_for_csv(data):
+    print(data)
     employee = data.get('employee', None)
     current_assignment = employee.get('currentAssignment', None)
-    # position = current_assignment.get('currentPosition', None)
+    position = None
```
Comment (cagbakwu) [inline talentmap_api/fsbid/services/client.py:315]: Added initial instance of position.

File: talentmap_api/fsbid/services/client.py [hunk 3]
```diff
@@ -309,9 +309,10 @@ def fsbid_clients_to_talentmap_clients(data):
 
 
 def fsbid_clients_to_talentmap_clients_for_csv(data):
+    print(data)
```
Comment (M00staff) [inline talentmap_api/fsbid/services/client.py:312]: 🤔

File: talentmap_api/fsbid/services/client.py [hunk 4]
```diff
@@ -326,7 +327,7 @@ def fsbid_clients_to_talentmap_clients_for_csv(data):
     suffix_name = f" {employee['per_suffix_name']}" if pydash.get(employee, 'per_suffix_name') else ''
     combined_location = f"{pos_location} ({position.get('pos_org_short_desc', None)})"
     cdo = data.get('cdos', None)
-
+    ted_conversion = dateFormat(current_assignment.get("asgd_etd_ted_date", None))
```
Comment (M00staff) [inline talentmap_api/fsbid/services/client.py:329]: what happens if `asgd_etd_ted_date` is None or a Null value..?

Comment (M00staff) [reply talentmap_api/fsbid/services/client.py:329]: not sure about python, but in JS `dateFormat` would prolly crash the page/throw an error 

you can test it out by just passing `None` to `dateFormat` 🤔

**Review by M00staff — APPROVED** [review]

**Pull request merged**